pkg/render/apiserver_test.go

```diff
@@ -54,9 +54,11 @@ var _ = Describe("API server rendering tests (Calico Enterprise)", func() {
 	var k8sServiceEp k8sapi.ServiceEndpoint
 
 	BeforeEach(func() {
+		var replicas int32 = 2
 		instance = &operatorv1.InstallationSpec{
-			Registry: "testregistry.com/",
```

Comment:
Please include a test specifying 1 replica and ensure that test case ensures no Affinity is specified.
Please also include checking affinity is set with more than 1 replica